lib/components/molecules/AtomicCarousel/AtomicCarousel.style.js

```diff
@@ -0,0 +1,26 @@
+import { css } from 'styled-components';
+
+const styles = css`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  .carousel-container {
+    display: flex;
+    align-items: center;
+  }
+
+  .atomic-dots {
```

Comment:
Why this can't be joined with above selector? 
.carousel-container, .atomic-dots